crates/bevy_winit/src/lib.rs

```diff
@@ -529,10 +492,19 @@ pub fn winit_runner(mut app: App) {
                                 window: window_entity,
                             });
                     }
-                    WindowEvent::KeyboardInput { ref input, .. } => {
-                        event_writers
-                            .keyboard_input
-                            .send(converters::convert_keyboard_input(input, window_entity));
+                    WindowEvent::KeyboardInput { ref event, .. } => {
+                        let keyboard_event =
+                            converters::convert_keyboard_input(event, window_entity);
+                        if let bevy_input::keyboard::Key::Character(c) = &keyboard_event.logical_key
+                        {
+                            if let Some(first_char) = c.chars().next() {
```

Comment:
I.. guess not, but sending the whole SmolStr within ReceivedCharacter seemed a big change when I was on it.

I can see multiple solutions:
- sending SmolStr.
- sending several ReceivedCharacter(char)
- splitting ReceivedCharacter and ReceivedMultipleCharacters ?

I thought this implementation was the closest to current main behaviour, so that might be OK as a first approach.
I'm writing that in "Follow Up" for the time being.